CommonTools/ParticleFlow/plugins/PFPileUp.cc

```diff
@@ -1,25 +1,100 @@
-#include "CommonTools/ParticleFlow/plugins/PFPileUp.h"
+#ifndef PhysicsTools_PFCandProducer_PFPileUp_
```

Comment:
I assume noboby will ever include this plugin: therefore this `#ifndef` -> `#define` -> `#endif` structure is not needed